spec/unit/network/http/handler_spec.rb

```diff
@@ -88,8 +93,7 @@ def respond(text)
       expect(res[:content_type_header]).to eq("application/json")
       expect(res_body["issue_kind"]).to eq(Puppet::Network::HTTP::Issues::RUNTIME_ERROR.to_s)
       expect(res_body["message"]).to eq("Server Error: the sky is falling!")
-      expect(res_body["stacktrace"].is_a?(Array) && !res_body["stacktrace"].empty?).to be_truthy
-      expect(res_body["stacktrace"][0]).to match("spec/unit/network/http/handler_spec.rb")
+      expect(res_body["stacktrace"]).to be_nil
```

Comment:
This line would probably look weird without the context of this ticket, probably worth adding a comment / link to ticket.